packages/camera/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: camera
 description: A Flutter plugin for getting information about and controlling the
   camera on Android and iOS. Supports previewing the camera feed and capturing images.
-version: 0.2.9
+version: 0.2.9+1
```

Comment:
Why not just 0.2.10?

Reply:
Following the guideline at https://www.dartlang.org/tools/pub/versioning#semantic-versions
it says that prior to 1.0.0: `...going to 0.1.2+1 indicates a change that doesn’t affect the public API.`